src/fortran_routing/Reservoir_Binding/Reservoirs/bind_hybrid.f90

```diff
@@ -18,16 +17,16 @@ END FUNCTION get_hybrid_handle
         SUBROUTINE free_hybrid(handle) BIND(C, NAME='free_hybrid')
             USE, INTRINSIC :: ISO_C_BINDING, ONLY: C_PTR, C_F_POINTER
             TYPE(C_PTR), INTENT(IN), VALUE :: handle
-            !TYPE(LP_Example), POINTER :: lp_ptr
             TYPE(persistence_levelpool_hybrid), POINTER :: hybrid_ptr
             CALL C_F_POINTER(handle, hybrid_ptr)
-            !DEALLOCATE(hybrid_ptr)
+            DEALLOCATE(hybrid_ptr)
```

Comment:
@hellkite500 The deallocation of the hybrid and the rfc ptrs work now. I think it was something else in the test failing previously that was not allowing these to properly execute. Thanks.